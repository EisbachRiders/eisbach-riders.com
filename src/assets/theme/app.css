html {
  height: 100%;
}

body {
  margin: 0;
  padding: 0;
  font-family: 'Raleway', sans-serif;
  background-color: #ffffff;
  overflow-x: hidden;
}

a {
  text-decoration: none;
}
