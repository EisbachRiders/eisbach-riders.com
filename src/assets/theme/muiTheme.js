import { createMuiTheme } from '@material-ui/core/styles'
import './App.css'
require('typeface-open-sans')
require('typeface-contrail-one')

const status = {
  white: '#ffffff',
  offWhite: '#F7F5F5',
  black: '#0b0c10',
  blue: '#1f2833',
  grey: '#c5c6c7',
  greyDk: '#404040',
  teal: '#66fcf1',
  tealDk: '#45a29e',
  green: '#66fca6',
  yellow: '#fcf166',
  red: '#e25b65',
}

const theme = createMuiTheme({
  palette: {
    type: 'light',
    primary: {
      light: status.teal,
      main: status.teal,
      dark: status.tealDk,
    },
    secondary: {
      light: status.blue,
      main: status.blue,
      dark: status.blue,
    },
    error: { main: status.red },
  },
  status,
  typography: {
    fontFamily: "'Open Sans', sans-serif",
    fontSize: 14,
  },
  overrides: {
    MuiTypography: {
      colorPrimary: {
        color: status.greyDk,
      },
      colorSecondary: {
        color: status.grey,
      },
      colorTextSecondary: {
        color: status.white,
      },
    },
  },
})

export default theme
