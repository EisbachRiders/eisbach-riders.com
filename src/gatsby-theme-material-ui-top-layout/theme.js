import { createMuiTheme } from "@material-ui/core/styles"
import "../app.css"
import "../../static/fonts/fonts.css"

// fin: "#00d7a2",
// finDk: "#00be8f",
// finLt: "#e2f9ef",
// greyLt2: "#eeeff0",
// greyLt: "#d5d9db",
// grey: "#818C93",
// greyBlue: "#f3f8fc",
// greenLt1: "#6dff97",
// greenLt2: "#00cf3c",
// cream: "#f7f5ee",

const color = {
  // for shop colors
  blue: "#004AB5",
  green: "#00942b",
  yellow: "#fce566",
  orange: "#ff8000",
  pink: "#ff0080",
  red: "#FF0012",
  redLt: "#ff4e5b",
  // theme colors
  white: "#ffffff",
  black: "#000000",
  almostBlack: "#363738", // typography
  textLight: "#838588",
  gray: "#ebedee",
  primaryBlue: "#0062f0",
  primaryBlueLt: "#3f8eff",
  gold: "#E6C55E",
  goldLt: "#f0dda0",
}

const theme = createMuiTheme({
  palette: {
    type: "light",
    primary: {
      light: color.goldLt,
      main: color.gold,
      dark: color.goldLt,
    },
    secondary: {
      light: color.primaryBlueLt,
      main: color.primaryBlue,
      dark: color.primaryBlueLt,
    },
    error: { main: color.red },
  },
  color,
  typography: {
    fontFamily: "'Raleway', sans-serif",
    h1: {
      letterSpacing: 2,
    },
    h2: {
      letterSpacing: 2,
    },
    h3: {
      letterSpacing: 2,
    },
    h4: {
      letterSpacing: 2,
    },
    h5: {
      letterSpacing: 2,
    },
    body1: {
      letterSpacing: 1.5,
      fontSize: 14,
    },
    body2: {
      letterSpacing: 1.5,
      fontSize: 12,
    },
    button: {
      fontFamily: "'Raleway', sans-serif",
      letterSpacing: 3,
    },
  },
  overrides: {
    MuiButton: {
      root: {
        padding: "9px 27px",
        borderRadius: 0,
      },
      contained: {
        boxShadow: "none",
      },
    },
  },
})

export default theme
