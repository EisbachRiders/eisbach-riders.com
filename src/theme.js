import { createTheme } from "@material-ui/core/styles"
import "./css/app.css"

// fin: "#00d7a2",
// finDk: "#00be8f",
// finLt: "#e2f9ef",
// grayLt2: "#eeeff0",
// grayLt: "#d5d9db",
// gray: "#818C93",
// grayBlue: "#f3f8fc",
// greenLt1: "#6dff97",
// greenLt2: "#00cf3c",
// cream: "#f7f5ee",

const color = {
  // for shop colors
  blue: "#004AB5",
  green: "#00942b",
  yellow: "#fce566",
  orange: "#ff8000",
  pink: "#ff0080",
  red: "#FF0012",
  redLt: "#ff4e5b",
  seaGlass: "#5F5850",
  // theme colors
  white: "#ffffff",
  black: "#0B0C10",
  blackLt: "#1F2833",
  gray: "#C5C6C7",
  // teal: "#66FCF1",
  teal: "#00d7a2",
  tealDk: "#45A29E",
  test: "#F7F680",
  cream: "#F1F0EB",
  grayLt: "#f7f7f9",
  blueGray: "#cbd3cf",
  steel: "#505D72",
}

const theme = createTheme({
  palette: {
    type: "light",
    primary: {
      light: color.tealDk,
      main: color.teal,
      dark: color.tealDk,
    },
    secondary: {
      light: "#9d9fa0",
      main: color.gray,
      dark: "#9d9fa0",
    },
    error: { main: color.red },
  },
  color,
  typography: {
    fontFamily: ["Raleway", "sans-serif"].join(","),
  },
  components: {
    MuiButton: {
      styleOverrides: {
        root: {
          padding: "9px 27px",
          borderRadius: 0,
        },
        contained: {
          boxShadow: "none",
        },
      },
    },
  },
})

export default theme
