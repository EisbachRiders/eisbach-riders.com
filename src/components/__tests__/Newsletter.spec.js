import React from 'react'
import { render } from 'react-testing-library'
import Newsletter from '../Newsletter'
import { ThemeProvider } from '@material-ui/styles'
import theme from '../../theme'

describe('Newsletter', () => {
  const mockProps = {
    t: () => '',
  }
  it('renders correctly', () => {
    const component = render(
      <ThemeProvider theme={theme}>
        <Newsletter {...mockProps} />
      </ThemeProvider>
    )
    expect(component).toMatchSnapshot()
  })
})
