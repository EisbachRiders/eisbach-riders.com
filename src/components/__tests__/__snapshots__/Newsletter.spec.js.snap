// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Newsletter renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Hook-containerNewsletter-122zkk4"
      >
        <h5
          class="MuiTypography-root-8 MuiTypography-h5-24 Hook-textNewsletter-kn9p3n"
        >
          newsletter.sectionTitle
        </h5>
        <p
          class="MuiTypography-root-8 MuiTypography-body1-17 Hook-subtitleNewsletter-f88uxn"
        >
          newsletter.message
        </p>
        <button
          class="MuiButtonBase-root-61 MuiButton-root-44 MuiButton-contained-52 MuiButton-containedPrimary-53 Hook-button-66oo39"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label-45"
          >
            newsletter.signUp
          </span>
          <span
            class="MuiTouchRipple-root-81"
          />
        </button>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Hook-containerNewsletter-122zkk4"
    >
      <h5
        class="MuiTypography-root-8 MuiTypography-h5-24 Hook-textNewsletter-kn9p3n"
      >
        newsletter.sectionTitle
      </h5>
      <p
        class="MuiTypography-root-8 MuiTypography-body1-17 Hook-subtitleNewsletter-f88uxn"
      >
        newsletter.message
      </p>
      <button
        class="MuiButtonBase-root-61 MuiButton-root-44 MuiButton-contained-52 MuiButton-containedPrimary-53 Hook-button-66oo39"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-label-45"
        >
          newsletter.signUp
        </span>
        <span
          class="MuiTouchRipple-root-81"
        />
      </button>
    </div>
  </div>,
  "debug": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
